futurenhs.app/types/page.d.ts

```diff
@@ -13,28 +13,29 @@ import {
     GroupsPageTextContent,
 } from '@appTypes/content'
 import { User } from '@appTypes/user'
+import { CmsContentPage, CmsContentBlock } from '@appTypes/cmsContent'
 
 export interface Page {
-    id: string
-    routes: Routes
-    etag?: string | Record<string, string>
-    themeId?: string
-    layoutId?: layoutIds
-    actions?: Array<actions>
-    csrfToken?: string
-    forms?: Record<string, FormConfig>
-    services?: Record<string, Service>
-    pagination?: Pagination
-    errors?: Array<Record<string>>
-    contentText?: GenericPageTextContent
-    contentPageId?: string
-    contentTemplateId?: string
-    contentTemplate?: Array<ContentBlock>
-    contentBlocks?: Array<ContentBlock>
-    user?: User
-    className?: string
-    pageTitle?: string
-    notifications?: Array<Notification>
```

Comment:
this needs to remain to set notifications on server side submissions

Reply:
Sorted, inadvertently removed